tests/integration/utils/utils.ts

```diff
@@ -0,0 +1,161 @@
+import { TezosToolkit } from '@taquito/taquito';
+import { exec as exec1 } from 'child_process';
+import fsPromises from 'fs/promises';
+import path from 'path';
+
+export const exec = (cmd: string, opts: Record<string, unknown> = {}): Promise<{ stdout: string; stderr: string }> =>
+	new Promise((resolve: Function, reject: Function) => {
+		exec1(cmd, opts, (err, stdout, stderr) => {
+			if (err) {
+				typeof err === 'string'
+					? reject({ message: err, stderr, stdout })
+					: reject(Object.assign({ message: stderr, stderr, stdout }, err));
+			} else {
+				resolve({
+					stdout,
+					stderr,
+				});
+			}
+		});
+	});
+
+export const sleep = (ms: number) => new Promise((resolve, _reject) => setTimeout(resolve, ms));
+
+export const generateTestProject = async (
+	projectPath: string,
+	packageNames: string[] = [],
+	localPackages: boolean = true,
+) => {
+	const targetDir = path.join('/tmp', projectPath);
+
+	const projectInit = await exec(`taq init ${targetDir}`);
+
+	await checkFolderExistsWithTimeout(targetDir);
+	await fsPromises.chmod(targetDir, 0o777);
+	await exec(`mv ${targetDir} ./${projectPath}`, { encoding: 'utf8' });
+
+	await checkFolderExistsWithTimeout(path.join('./', projectPath, 'package.json'));
+	await fixCorePlugin(projectPath);
+
+	await installDependencies(projectPath, packageNames, localPackages);
+
+	return projectInit;
+};
+
+const fixCorePlugin = async (projectPath: string) => {
+	// TODO: We need to come back to this.
+	//
+	// The core plugin gets installed automatically when a taqueria project is initialized.
+	// However, the version of this plugin is pulled from npmjs.com, which might be using
+	// an older version of the SDK, and that older version of the SDK might be using a version
+	// of @taqueria/protocol package that is incompatible with the development version of the
+	// taq binary, which is the version that was meant to be used by the E2E tests.
+	//
+	// To fix this, we need to do the following:
+	//
+	// 1) Uninstall the core plugin pulled from npmjs.com
+	await exec('npm uninstall @taqueria/plugin-core', { cwd: projectPath });
```

Comment:
Don't need this as CORE is no longer installed on `taq init` without a `--workflow` option

Reply:
The revised tests are not dependant on the utils file, which will be eventually dropped

Reply:
But maybe this is an excellent opportunity to clean it up for the other tests not revised yet. I will try it.